README.md

```diff
@@ -63,6 +63,8 @@ cv2.waitKey(0)
 
 You can install this package locally from source. Make sure you have all necessary dependencies setup (see Installation section above). 
 
+**NOTE:** When using `venv`, the environment should be created without the `--system-site-packages` flag; otherwise building might fail.
```

Comment:
```suggestion
**NOTE:** When using a Python virtual environment, it should be created without the `--system-site-packages` flag; otherwise building might fail.
```